### PR TITLE
Making Content Great Again batch 2

### PR DESCRIPTION
**Pull request by hod-alpert**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

##related issue: 
https://github.com/demisto/etc/issues/23712

## Description
fixing content files so that they pass validation- batch 2

### CODE REVIEW

File: Packs/Anomali_ThreatStream/TestPlaybooks/playbook-Anomali_ThreatStream_Test.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-id: Anomali_ThreatStream_Test
+id: Anomali ThreatStream Test
```
Comment (Itay4) [inline Packs/Anomali_ThreatStream/TestPlaybooks/playbook-Anomali_ThreatStream_Test.yml:1]: dont we need to update this in `conf.json` as well ?

File: Packs/BitDam/TestPlaybooks/playbook-BitDam_Test.yml [hunk 2]
```diff
@@ -33,11 +34,12 @@ tasks:
     task:
       id: def55e04-3ec5-4160-88d6-edd57d1ddb6b
       version: -1
-      name: Scan File - BitDam
+      name: Detonate File - BitDam
```
Comment (Itay4) [inline Packs/BitDam/TestPlaybooks/playbook-BitDam_Test.yml:37]: why the change here?

File: Packs/CVESearch/TestPlaybooks/playbook-CVE-Search-Test.yml [hunk 3]
```diff
@@ -1,10 +1,10 @@
 id: CVE Search v2 - Test
 version: -1
 name: CVE Search v2 - Test
```
Comment (Itay4) [inline Packs/CVESearch/TestPlaybooks/playbook-CVE-Search-Test.yml:3]: please make sure this test still runs and passes as expected

File: Tests/conf.json [hunk 4]
```diff
@@ -2816,6 +2816,7 @@
     },
     "skipped_integrations": {
         "_comment1": "~~~ NO INSTANCE ~~~",
+        "ArcSight Logger": "Issue 24303",
```
Comment (Itay4) [inline Tests/conf.json:2819]: are you synced with @ShahafBenYakir on this ?

File: Packs/Anomali_ThreatStream/TestPlaybooks/playbook-Anomali_ThreatStream_Test.yml (hunk 1 above)
Comment (hod-alpert) [reply Packs/Anomali_ThreatStream/TestPlaybooks/playbook-Anomali_ThreatStream_Test.yml:1]: This is a mistake, we decided I won't touch name != id issues, I'll fix that

File: Packs/CVESearch/TestPlaybooks/playbook-CVE-Search-Test.yml (hunk 3 above)
Comment (hod-alpert) [reply Packs/CVESearch/TestPlaybooks/playbook-CVE-Search-Test.yml:3]: Did that, It was my bad to make the id's uuids to begin with

File: Tests/conf.json (hunk 4 above)
Comment (hod-alpert) [reply Tests/conf.json:2819]: Yup

File: Packs/CVESearch/TestPlaybooks/playbook-CVE-Search-Test.yml (hunk 3 above)
Comment (Itay4) [reply Packs/CVESearch/TestPlaybooks/playbook-CVE-Search-Test.yml:3]: so it passes?

Comment (hod-alpert) [reply Packs/CVESearch/TestPlaybooks/playbook-CVE-Search-Test.yml:3]: Yup

**Comment by Itay4** [comment]
build fails due to validations apporved to be skipped
merging
